Auth0/SafariWebAuth.swift

```diff
@@ -101,8 +102,9 @@ class SafariWebAuth: WebAuth {
         return self
     }
 
-    func useLegacyAuthentication() -> Self {
+    func useLegacyAuthentication(withStyle style: UIModalPresentationStyle = .fullScreen) -> Self {
```

Comment:
as this param is required and must be specified when used, the default is not required.

Reply:
If I remove the default value then previous versions of this method won't compile. I'm happy to remove the default if you want users to update their code to the default value manually.

Reply:
Default value has been removed.

Reply:
The previous version will work as your Public extension is the code that allows facilitates that.

Reply:
Apologies, you're correct, I had to run this in Xcode as my mental compiler wasn't running properly. I had mixed up the use of the `WebAuth` Protocol vs the `SafariWebAuth` object.

The use case I was thinking about was standard login e.g. Which will be fine as `.webAuth` returns the Protocol and the Public Extension maintains the Public API and Backwards compatibility.

```swift
  Auth0.webAuth()
            .logging(enabled: true)
            .useLegacyAuthentication()
            .start(onAuth)
        }
```

In tests the `SafariWebAuth` is used directly which does not have any default. So please restore :)